content/images.md

```diff
@@ -3,17 +3,6 @@ title: faucet-pipeline-images
 faucet-pipeline-images offers image optimization and transformation.
 It supports JPG, PNG, SVG, WebP and AVIF.
 
-Currently, you need to **activate the plugin manually**:
-
-```js
-module.exports = {
-    // ...
-    plugins: [
-        require("faucet-pipeline-images")
```

Comment:
FWIW: I *think* `plugins: ["faucet-pipeline-images"]` should have worked as well